### ISSUE TITLE
Menu auto complete helper

### ISSUE THREAD

**Issue by jcubic** [jcubic/jquery.terminal#511]
### I have idea for a new feature for jQuery Terminal

Add helper from https://codepen.io/jcubic/pen/MJyYEx to terminal file to simplify creating of auto complete menu:

it work the same as normal completion function or array, but call helper that wrap the options object:

```javascript
$('body').terminal($.noop, $.terminal.menu_autocomplete({
  completion: function(string) {
      return fetch('completion.php', {string: string});
  }
});
```

consider API that allow the stacks of wrappers? How this should look like?

**Comment by jcubic** [comment]
Another potential experimental wrapper would be echo without newline, the code is hack but it can be put into external file.

**Comment by jcubic** [comment]
I've decided that there will be no API for this, you just need to include the file and the code will do monkey patch.

**Comment by jcubic** [comment]
Echo newline file added it work the same as autocomplete_menu just need to include the file. The code also handle prompt as function. But still both files need some tests to be written.

**Comment by jcubic** [comment]
released in 2.8.0
